Options.py

```diff
@@ -1256,6 +1260,8 @@ def as_dict(self, *option_names: str, casing: str = "snake") -> typing.Dict[str,
                 value = getattr(self, option_name).value
                 if isinstance(value, set):
                     value = sorted(value)
+                if toggles_as_bools and issubclass(type(self).type_hints[option_name], Toggle):
```

Comment:
I think an elif is more appropriate here since a Toggle can't have a set as its value.